pkg/handler/processor/processor.go

```diff
@@ -53,6 +53,8 @@ type DocumentType string
 // Document* is the enumerables of DocumentType
 const (
 	DocumentITE6SLSA    DocumentType = "SLSA"
+	DocumentITE6CREV    DocumentType = "CREV"
+	DocumentITE6REVIEW  DocumentType = "REVIEW"
```

Comment:
I think we can skip these , let `DocumentITE6Unknown` is good, maybe we rename it to `DocumentITE6Generic` instead

Reply:
If we do this. How will the ingestor know how to parse? As we want to capture the payload associated with the predicate.

Reply:
i think it would flatten each into a dictionary - at least for now, until we establish the pred dictionary in the future. The fields can just be all included for now for anything that isn't SLSA, so the trade off would be that for now, it only points to the subject.